javacutil/src/main/java/org/checkerframework/javacutil/AnnotationUtils.java

```diff
@@ -684,9 +684,13 @@ public static boolean hasElementValue(AnnotationMirror anno, CharSequence elemen
      * Get the element with the name {@code elementName} of the annotation {@code anno}. The result
      * is expected to have type {@code expectedType}.
      *
-     * <p>For elements of array type, use {@code getElementValueArray} instead.
+     * <p>If the return type is an array, use {@link #getElementValueArray} instead.
      *
-     * <p>For elements of enum type, use {@code getElementValueEnum} instead.
+     * <p>If the return type is an enum, use {@link #getElementValueEnum} instead.
+     *
+     * <p>If possible, it is more efficient to use {@code
```

Comment:
Just for my own understanding, this comment is true for all the `getElementValue*` method right?  Also, if the annotation only has one element, is it still more efficient to call `anno.getElementValues().get(someElement).getValue()`?

Reply:
Yes to both questions.  I updated the comment.